packages/documentation/src/layouts/module-components.js

```diff
@@ -15,13 +15,26 @@ const MyCodeComponent = ({ children }) => (
 );
 
 export default class MDXRuntimeTest extends Component {
+  sortedReqiuredProps() {
```

Comment:
@rianfowler Because this method gets directly invoked in the template it should already be bound to this, so I don't think we need to bind it. It works without needing to bind it.